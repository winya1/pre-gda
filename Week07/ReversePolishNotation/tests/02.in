1
1 2 +
1 2 + 3 *
1 2 + 3 * 1 /
1 2 + 3 * 4 5 - +
2 3 4 + + 
15 7 1 1 + - / 3 * 2 1 1 + + -
1 2 + 3 /
1 2 -
2 1 -
