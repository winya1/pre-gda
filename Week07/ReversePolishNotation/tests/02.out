1.0
3.0
9.0
9.0
8.0
9.0
5.0
1.0
-1.0
1.0
